PyPortal_Thingiverse/thingiverse.py

```diff
@@ -36,8 +37,8 @@
         print("Response is", response)
         pyportal.set_background(None)
         image_url = response[2].replace('_thumb_medium.', '_display_large.')
-        pyportal.wget(adafruit_pyportal.IMAGE_CONVERTER_SERVICE+image_url, "/cache.bmp")
-        pyportal.set_background("/cache.bmp")
+        pyportal.wget(pyportal.image_converter_url(image_url, 320, 240), "/sd/cache.bmp")
```

Comment:
not sure if it's consistent across URLs, but I had trouble with wget never returning if I didn't manually set the chunk_size low: 

```python
        pyportal.wget(pyportal.image_converter_url(DATA_SOURCE, 320, 240), "/sd/cache.bmp", chunk_size=512)
```

Reply:
yes SD cards require a smaller chunk (due to a bug) - @martymcguire want to fix?

Reply:
added `chunk_size=512`. seems to work okay!